jaxley_mech/channels/benav12.py

```diff
@@ -0,0 +1,214 @@
+from typing import Dict, Optional, Union
+
+import jax.debug
+import jax.numpy as jnp
+from jax.lax import select
+from jaxley.channels import Channel
+from jaxley.solver_gate import exponential_euler, save_exp, solve_gate_exponential, solve_inf_gate_exponential
+
+from jaxley_mech.solvers import SolverExtension
+
+META = {
+    "cell_type": "bipolar cell",
+    "species": ["Human Embryonic Kidney Cells"],
+    "reference": "Benav, H. (2012)",
+    "doi": "http://hdl.handle.net/10900/46043",
+    "note": "The model is using the reduced voltage not the membrane potential. Therefore, the resting potential has to be given as a parameter."
+}
+
+
+class Ca_T(Channel):
+    """ Transient type of Calcium Channel """
+
+    def __init__(self, 
+                 v_rest_global: float,
+                 name: Optional[str] = None):
+        
+        self.current_is_in_mA_per_cm2 = True
+                
+        super().__init__(name)
+        self.channel_params = {
+            # To match the experimental data, I had to adapt the conductance
+            f"{self._name}_gCa_T": 0.03634,  # S/cm^2
+
+
+            # This is the calcaultion from the dissertation of H. Benav:
+            # E_{Ca_T} = Neernst Ca++ * 45/100 + E_K * 55/100
+            # E_K is taken from usui96 of Kv channel
+            # E_{Ca_T} = 132.65mV ×0.45 + (-58mV) * 0.55 = 27.5mV
+            # Equilibrium potential for calcium:
+            "eCa": 27.5, # mV  

```

Comment:
Generally speaking, intracellular Ca concentration is low, so even a small change of Ca will change the eCa. `Nernst[Ca++]` should be dynamic. 

Put the eCa to channel_states, you can use `CaNernstReversal` from `fohlmeister97` to do it.